app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+// -- Headers ----------------------------------------------------------------------
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Roobert-Bold', 'Times';
+    font-style: normal;
+    font-weight: bold;
+    color: $blue_primary;
+}
+
+// ---------------------------------------------------------------------------------
+
+
+// -- Paragraphs -------------------------------------------------------------------
+p{
+    font-family: 'Roobert-Regular', sans-serif!important;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
```

Comment:
There is no need to declare a value for `font-size` here since it will be inherited from the existing styles. if we do though can we use `rem` for units?